http/relay/relay_test.go

```diff
@@ -21,14 +24,55 @@ func TestRelay(t *testing.T) {
 	c := &up.Config{}
 	assert.NoError(t, c.Default(), "default")
 
-	h, err := New(c)
-	assert.NoError(t, err, "init")
+	var h http.Handler
+	newH := func(t *testing.T) {
+		localH, err := New(c)
+		assert.NoError(t, err, "init")
+		h = localH
+	}
```

Comment:
I was getting weird cross-talk between my tests if I didn't do this, this is definitely slowing the tests down a smidge - so I can be a little bit more scientific about it if you like.